### ISSUE TITLE
Book 05: Sentence 051

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#34]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8b1a040a-8631-4cf4-9720-5a4c3d3d7e21)

```
# sent_id = Book_05_-_The_Omens.txt:51
# text = Ic chiquacen capitulo, vncan mjtoa: in tetzaujtl catca, injc motetzaujaia: in jquac aca qujttaia, cocamatl in teixpan tlaujltequja.
# text[norm] = Ic chicuacen capítulo, oncan mihtoa: in tetzahuitl catca, inic motetzahuiaya: in ihcuac aca quittaya, cozamatl in teixpan tlahuiltequiya.
# text[orig] = Icchiquacen·capitulo·,·vncan·mj¶toa·:·intetzaujtl·catca·,·injc·motetzauj¶aia·:·injquac·aca·qujttaia·,·cocamatl·in¶teixpan·tlaujltequja·.
# text[osp] = El sexto capitulo, tracta del mal aguero que tomauan: quando vian, que la comadreja, o mostolilla atrauesaua por delante dellos, quando yvan por el camjno, o por la calle.
# references = A&D/170:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 62.50%
1	Ic	ic	ADV	_	_	3	amod	_	_
2	chiquacen	chicuacen	NUM	_	_	1	nummod	_	_
3	capitulo	capítulo	NOUN	_	State=Abs	6	obl	_	_
4	,	,	PUNCT	_	_	6	punct	_	_
5	vncan	oncan	ADV	_	_	6	advmod	_	_
6	mjtoa	ihtoa	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
7	:	:	PUNCT	_	_	9	punct	_	_
8	in	in	DET	_	_	9	mark	_	_
9	tetzaujtl	tetzahuitl	NOUN	_	State=Abs	6	csubj	_	_
10	catca	catqui	AUX	_	Aspect=Imp|Number[subj]=Sing|Person[subj]=3|Tense=Past|VerbForm=Fin	9	cop	_	_
11	,	,	PUNCT	_	_	13	punct	_	_
12	injc	inic	SCONJ	_	_	13	mark	_	_
13	motetzaujaia	tetzahuia	VERB	_	_	9	conj	_	_
14	:	:	PUNCT	_	_	18	punct	_	_
15	in	in	SCONJ	_	_	18	mark	_	_
16	jquac	ihcuac	ADV	_	_	18	advmod	_	_
17	aca	aca	PRON	_	_	18	nsubj	_	_
18	qujttaia	itta	VERB	_	Aspect=Imp|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Past|VerbForm=Fin	13	advcl	_	_
19	,	,	PUNCT	_	_	20	punct	_	_
20	cocamatl	cozamatl	NOUN	_	State=Abs	18	obj	_	_
21	in	in	SCONJ	_	_	23	mark	_	_
22	teixpan	ixpan	NOUN	_	Animacy[psor]=Hum|NounType=Relat|Person[psor]=3|State=Pos	23	obl	_	_
23	tlaujltequja	tlahuiltequi	VERB	_	Aspect=Imp|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	20	acl	_	_
24	.	.	PUNCT	_	_	6	punct	_	_
```

**Comment by ftyers** [comment]
This is similar to #14, but we should check it and make sure that:
- `conj` is right for *inic motetzahuiaya*
- the structure of *quittaya, cozamatl in teixpan tlahuiltequia* is right

**Comment by Lguyogiro** [comment]
I think _quittaya_ should be `csubj` of _motetzahuiaya_, since the seeing is what was considered portentous.

**Comment by Lguyogiro** [comment]
If we make quittaya the `csubj`, then I think it looks good, esp. wrt the question of `conj`, since then both _in tetzahuitl catca_ and _motetzahuiaya_ have the same subject.

**Comment by ftyers** [comment]
You mean *in ihcuac aca quittaya* is the `csubj` of *in tetzahuitl catca* and *inic motetzahuiaya* ?  :exploding_head:  Can you throw that in a tree?

**Comment by Lguyogiro** [comment]
This is what I was thinking

![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/aeeae3ac-af80-43d8-9ab1-f6c0dc1ad7c5)

but I now see the interpretation in the original tree, and it also makes sense. I understood this as, the following clause, e.g. the crossing of the animal in front of someone, was taken as a portent (i.e. a pseudo-passive reading of the reflexive).

**Comment by ftyers** [comment]
What is the motivation behind `conj` between *quittaya* and *cozamatl* ? 

In that case, wouldn't *in ihcuac aca quittaya* be `csubj` to *tetzahuitl catca* not to *motetzahuiaya*? (although not arguing for this analysis) ... I think both maybe work?

**Comment by Lguyogiro** [comment]
> What is the motivation behind conj between quittaya and cozamatl ?

This is my mistake, changing it to ´obj´ now

**Comment by Lguyogiro** [comment]
> In that case, wouldn't in ihcuac aca quittaya be csubj to tetzahuitl catca not to motetzahuiaya? (although not arguing for this analysis) ... I think both maybe work?

Either of these would work for me. 

I guess deciding whether its `csubj(motetzahuia, <clause>)` or  `advcl(motetzahuia, <clause>)` will have big repercussions, as I'm pretty sure this construction is found up and down this book.

**Comment by ftyers** [comment]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/d64e8d53-de0b-47f0-ae39-24e63558c0ba)

Is this the suggestion? Also... Are there `expl` or 0-place verbs in Nahuatl? (well, aside from e.g. *quiyohui*)

Perhaps we can break it down by `VERB`:

* What is the subject of *mihtoa*?
   - ?
* What is the subject of *tetzaujtl catca*?
   - ?
* What is the subject of *motetzahuiaya*?
   - ?
* What is the subject of *qujttaia*?
   - *aca* "someone"
* What is the subject of *tlahuiltequiya*? 
   - *cozamatl* "weasel" (from the relative clause)

**Comment by Lguyogiro** [comment]
> Is this the suggestion? Also... Are there expl or 0-place verbs in Nahuatl? (well, aside from e.g. quiyohui)

No, I just meant in terms of understanding the English glosses for the two interpretations: when _quittaya_ heads the clausal subject of _motetzahuia_, the gloss is something like the "occurrence of _clause_ was considered a portent". If _quittaya_ heads an adverbial clause modifying _motetzahuiaya_, the gloss is like "it/something is considered a portent when _clause_" right?

**Comment by ftyers** [comment]
> > Is this the suggestion? Also... Are there expl or 0-place verbs in Nahuatl? (well, aside from e.g. quiyohui)
> 
> No, I just meant in terms of understanding the English glosses for the two interpretations: when _quittaya_ heads the clausal subject of _motetzahuia_, the gloss is something like the "occurrence of _clause_ was considered a portent". If _quittaya_ heads an adverbial clause modifying _motetzahuiaya_, the gloss is like "it/something is considered a portent when _clause_" right?

Yes, that is my understanding... But are there any diagnostics for distinguishing the two?

**Comment by Lguyogiro** [comment]
re the subject of motetzahuiaya: I was interpreting this as "The passive notion expressed by an active voice VNC" with the reflexive pronoun.

![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/9dff4c13-069e-4154-af43-161c1ff58137)

If _quittaya_ heads an adverbial clause, I'm not sure what the subject of _motetzahuiaya_ is.

**Comment by Lguyogiro** [comment]
> What is the subject of mihtoa?

I think our analysis of the copular phrase _in tetzahuitl catca_ as the subject is correct (see reflexive for expressing passive excerpt above from Andrews).

> What is the subject of tetzaujtl catca?
> What is the subject of motetzahuiaya?

My interpretation, is that the subject of both of these is the portent, and that the portent is represented by the clause headed by _quittaya_. i.e. the subject of "it was a portent" and "it was considered a portent" is "seeing a weasel..."

**Comment by ftyers** [comment]
> > What is the subject of mihtoa?
> 
> I think our analysis of the copular phrase _in tetzahuitl catca_ as the subject is correct (see reflexive for expressing passive excerpt above from Andrews).
> 
> > What is the subject of tetzaujtl catca?
> > What is the subject of motetzahuiaya?
> 
> My interpretation, is that the subject of both of these is the portent, and that the portent is represented by the clause headed by _quittaya_. i.e. the subject of "it was a portent" and "it was considered a portent" is "seeing a weasel..."

That's essentially what we have in https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/issues/34#issuecomment-1912891249, e.g. the whole `advcl` is the clausal subject of both *tetzahuitl catca* and of *motetzahuiaya*.

**Comment by Lguyogiro** [comment]
> That's essentially what we have in https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/issues/34#issuecomment-1912891249, e.g. the whole advcl is the clausal subject of both tetzahuitl catca and of motetzahuiaya.

Yes, and I am good with that analysis.

**Comment by ftyers** [comment]
Merged in 2803692.
